drivers/clock_control/clock_control_nrf.c

```diff
@@ -79,6 +79,10 @@ struct nrf_clock_control_config {
 static atomic_t hfclk_users;
 static uint64_t hf_start_tstamp;
 static uint64_t hf_stop_tstamp;
+#if CONFIG_CLOCK_CONTROL_NRF_K32SRC_SYNTH
```

Comment:
This should be `#if defined()`, since the symbol can be either defined to 1 or undefined.